MANIFEST.md

```diff
@@ -1,16 +1,14 @@
 title: The STELLARSTATION OOT Module
-brief: Short description of gr-stellarstation
-tags: # Tags are arbitrary, but look at CGRAN what other authors are using
+brief: gr-stellarstation is an out-of-tree GNURadio module that provides blocks for easily integrating with the Stellarstation API in GNURadio flowgraphs.
+tags:
   - sdr
+  - satellites
 author:
-  - Author Name <authors@email.address>
+  - Infostellar, Inc. <authors@email.address>
```

Comment:
You can omit ", Inc." here.
Also I believe authors@email.address should be replaced to something else. You can use `support@istellar.jp` .